src/Verbs/InitCommand.cs

```diff
@@ -0,0 +1,259 @@
+        {
+            var path = string.IsNullOrWhiteSpace(options.Folder) ? "." : options.Folder;
+
+            var currentFolder = new DirectoryInfo(path);
+
+            var packageFile = GetPackageFile(options.Folder);
+
+            var setup = new PackageSetup();
+
+            Console.WriteLine(Resources.Init_Header);
+            Console.WriteLine();
+
+            // gather all the user input
+
+            setup.Name = GetUserInput(Resources.Init_PackageName, Path.GetFileName(currentFolder.Name));
+
+            setup.Description = GetUserInput(Resources.Init_Description, Defaults.Init_Description);
+
+            setup.Version = GetVersionString(Resources.Init_Version, Defaults.Init_Version);
+
+            setup.Url = GetUserInput(Resources.Init_Url, Defaults.Init_Url);
+
+            setup.UmbracoVersion = GetVersionString(Resources.Init_UmbracoVersion, Defaults.Init_UmbracoVersion);
+
+            setup.Author = GetUserInput(Resources.Init_Author, Environment.UserName);
+
+            setup.Website = GetUserInput(Resources.Init_Website, Defaults.Init_Website);
+
+            setup.Licence = GetUserInput(Resources.Init_Licence, Defaults.Init_Licence);
+
+            // play it back for confirmation
+            Console.WriteLine();
+            Console.WriteLine(Resources.Init_Confirm, packageFile);
+
+            var node = MakePackageFile(setup);
+
+            Console.WriteLine(node.Element("info").ToString());
+
+            // confirm 
+
+            var confirm = GetUserInput(Resources.Init_Prompt, Defaults.Init_Prompt).ToUpper();
+            if (confirm[0] == 'Y')
+            {
+                // save xml to disk.
+
+                node.Save(packageFile);
+                Console.WriteLine(Resources.Init_Complete);
+                Environment.Exit(0);
+            }
+            else
+            {
+                Environment.Exit(1);
+            }
+        }
+
+        /// <summary>
+        ///  Make a package xml from the options
+        /// </summary>
+        /// <param name="options">options enterd by the user</param>
+        /// <returns>XElement containing the package.xml info</returns>
+        private static XElement MakePackageFile(PackageSetup options)
+        {
+            var node = new XElement("umbPackage");
+
+            var info = new XElement("info");
+
+            var package = new XElement("package");
+            package.Add(new XElement("name", options.Name));
+            package.Add(new XElement("version", options.Version));
+            package.Add(new XElement("iconUrl", ""));
+            package.Add(new XElement("licence", options.Licence,
+                new XAttribute("url", GetLicenceUrl(options.Licence))));
+            
+            package.Add(new XElement("url", options.Url));
+            package.Add(new XElement("requirements",
+                new XAttribute("type", "strict"),
+                new XElement("major", 8),
```

Comment:
Wondering if we should ask for the desired version? 🤔

Reply:
yes we can (and default it to v8)

Reply:
turns out we do, we where just not setting this value with it. fixed now